test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -1968,19 +1980,20 @@ static int destroy_all_threshold_profiles(void)
 
 static int destroy_all_wred_profiles(void)
 {
-	odp_tm_wred_t wred_profile;
+	odp_tm_wred_t wred_prof;
 	uint32_t      idx, color;
 	int           rc;
 
 	for (idx = 0; idx < NUM_WRED_PROFILES; idx++) {
 		for (color = 0; color < ODP_NUM_PKT_COLORS; color++) {
-			wred_profile = wred_profiles[idx][color];
-			if (wred_profile != ODP_TM_INVALID) {
-				rc = odp_tm_wred_destroy(wred_profile);
-				if (rc != 0) {
+			wred_prof = wred_profiles[idx][color];
+			if (wred_prof != ODP_TM_INVALID) {
+				rc = odp_tm_wred_destroy(wred_prof);
+				if (rc == 0) {
```

Comment:
Seems accidental change: rc != 0 --> rc == 0

Function return value is 0 on success.

Reply:
Will fix in V3